sdk/eventhub/azure-eventhubs/azure/eventprocessorhost/azure_storage_checkpoint_manager.py

```diff
@@ -37,6 +37,7 @@ class AzureStorageCheckpointLeaseManager(AbstractCheckpointManager, AbstractLeas
      will be used.
     :param str lease_container_name: The name of the container that will be used to store
      leases. If it does not already exist it will be created. Default value is 'eph-leases'.
+    :param str storage_blob_prefix: A string to prefix the lease storage path, default empty.
```

Comment:
I made the (wise?  foolish? :P) decision of polling individuals who had never touched this before to get a clean set of eyes on how intuitive the docs were.  As a result the docstrings were heavily modified and bolstered.  I really like how they go together now but I'm a wee bit concerned about length;  If I could be so greedy as to request your sanity check of the changes in terms of an "oh god what" would appreciate it.  (Additionally because we ended up doing less of a "examples everywhere" approach and more of a "unify the terms and concepts so that each makes sense in isolation" approach; but I'm torn between whether this could in some way be done more concisely with examples.  Will be thinking this over myself to not bikeshed you further, if you aren't sold on these.)